tests/test_unittest/test_gtscript_frontend.py

```diff
@@ -537,3 +542,41 @@ def definition_bw(
                 interval = definition_ir.computations[i].interval
                 assert interval.start.offset == axis_bound[0]
                 assert interval.end.offset == axis_bound[1]
+
+
+class TestNativeFunctions:
+    def test_simple_call(self):
```

Comment:
I miss tests mixing native functions and arithmetic operators, could you add them?

Reply:
done